src/clojure/nrepl/cmdline.clj

```diff
@@ -111,12 +111,14 @@ Exit:      Control+D or (exit) or (quit)"
           (let [input (read *in* false 'exit)]
             (if (done? input)
               (clean-up-and-exit 0)
-              (do (doseq [res (nrepl/message session {:op "eval" :code (pr-str input)})]
-                    (when (:value res) (value (:value res)))
-                    (when (:out res) (out (:out res)))
-                    (when (:err res) (err (:err res)))
-                    (when (:ns res) (set! *ns* (create-ns (symbol (:ns res))))))
-                  (recur)))))))))
+              (let [code-str (binding [*print-meta* true]
```

Comment:
I think you might want to add some comment here for posterity's sake.

Reply:
Done